controllers/util/storagecluster.go

```diff
@@ -0,0 +1,68 @@
+package util
+
+import (
+	"context"
+	"fmt"
+
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
+
+	nbv1 "github.com/noobaa/noobaa-operator/v5/pkg/apis/noobaa/v1alpha1"
+	rookCephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type AvailableServices struct {
+	Rbd    bool
+	CephFs bool
+	Nfs    bool
+	Mcg    bool
+}
+
+func GetAvailableServices(
+	ctx context.Context,
+	cl client.Client,
+	storageCluster *ocsv1.StorageCluster,
+) (*AvailableServices, error) {
```

Comment:
These are only relevant for the storageconsumer controller? why are we pushing them to the util package?